src/wand/magick.rs

```diff
@@ -650,7 +649,7 @@ impl MagickWand {
     }
 
     /// Extracts pixel data from the image as a vector of 0..255 values defined by `map`.
-    /// See https://www.imagemagick.org/api/magick-image.php#MagickExportImagePixels for more information.
```

Comment:
`www.imagemagick.org` for doc purpose looks like not working now, that's why I changed this line.